mmaction/datasets/rawframe_dataset.py

```diff
@@ -65,36 +67,49 @@ def __init__(self,
                  data_prefix=None,
                  test_mode=False,
                  filename_tmpl='img_{:05}.jpg',
+                 with_offset=False,
                  multi_class=False,
                  num_classes=None,
                  modality='RGB'):
+        self.filename_tmpl = filename_tmpl
+        self.with_offset = with_offset
         super().__init__(ann_file, pipeline, data_prefix, test_mode,
                          multi_class, num_classes, modality)
-        self.filename_tmpl = filename_tmpl
 
     def load_annotations(self):
         """Load annotation file to get video information."""
         video_infos = []
         with open(self.ann_file, 'r') as fin:
             for line in fin:
                 line_split = line.strip().split()
-                if self.multi_class:
+                item = {}
+                idx = 0
+                frame_dir = line_split[idx]
+                if self.data_prefix is not None:
+                    frame_dir = osp.join(self.data_prefix, frame_dir)
+                item['frame_dir'] = frame_dir
+                idx += 1
+                if self.with_offset:
+                    item['offset'] = int(line_split[idx])
+                    item['total_frames'] = int(line_split[idx + 1])
+                    idx += 2
+                else:
+                    item['total_frames'] = int(line_split[idx])
+                    idx += 1
+                label = [int(x) for x in line_split[idx:]]
```

Comment:
replace idx in line 95 and line 98 with `label_start_index=3` and `label_start_index=2`

Reply:
plz refer to previous comments